projects/ui/src/lib/components/ui-list/ui-list.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { TimeagoModule } from 'ngx-timeago'
```

Comment:
I don't think we need this module here :)

Reply:
The import NgModule is necesary